third_party/packages/flutter_svg/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.15
+
+* Changes internal `File` implementation in favor of `package:file`'s `File` implementation.
```

Comment:
It's not internal, it's part of the public interface of the package. Changing a type in the public interface is a breaking change that would require a major version bump.

Reply:
Removed, "internal", good catch!

Reply:
Removing the word internal doesn't address the primary problem, which is that this this is a *breaking change*, which is being versioned as if it weren't, and isn't prominently described as such following our CHANGELOG style guide.

Reply:
Apologies, I was multitasking and didn't read through the entire comment. You are correct, since an `io.File` instance is not of type `file.File` it would be a breaking change.

I would also like to avoid a breaking change but would want to improve the current DX around the .file constructor for Web.

I'm going to close this PR for now and if I have any bandwidth I will open a related issue. 

Note that, in theory the `.file` constructor is syntactic sugar that the package offers, since the same can be achieved with other `ByteLoaders`; for example by reading the File content first (either bytes or String) and then using an `SvgBytesLoader` or `SvgStringLoader`. Thus, the package could remove the need of `File` entirely.